README.md

```diff
@@ -40,10 +40,13 @@ AbsaOSS/k3d-action defines several input attributes and two outputs:
 
 - `args` (Optional) list of k3d arguments defined by [k3d command tree](https://k3d.io/usage/commands/)
 
+- `k3d-version` (Optional) version of k3d. If not set, will be used version from mapping below.
+
 ### Version mapping
 
 Implementation of additional features brings complexity and sometimes it may happen that extra feature is broken in special cases.
-To prevent potential issues, the `k3d` version is fixed according to the mapping below:
+Because of that, `k3d-action` has fixed default, tested against compatibility `k3d` version.
+Starting from k3d-action v2.1.0 users can explicitly set k3d version via `k3d-version` input e.g. `k3d-version: v5.2.2` otherwise k3d uses default version according to the mapping above.
```

Comment:
Move this paragraph below the table, over `For further k3s details see:...`. We already have very similar message there.

Sorry for confusions, thanks.

Reply:
Yeah I see it now, I was wondering what order of this notices should be - more natural is to have notice about k3d version above k3s version because dependencies of projects are k3d-action -> k3d -> k3s , but in the other hand setting up k3s version looks like more often scenario then setting up k3d version. I've used first order but if you want, I can replace it.